src/fontra/client/core/ui-list.js

```diff
@@ -55,6 +57,21 @@ export class List {
     this.container.classList.add("empty");
   }
 
+  includeAddRemoveButtons(addButtonCallback, removeButtonCallback) {
```

Comment:
I'm not sure this should be part of the ui-list module. Let's discuss live.

Reply:
I moved it to the editor.js to be configured all there